src/components/NavBar.css

```diff
@@ -0,0 +1,32 @@
+.NavBar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 75%;
```

Comment:
Why does this need a width?

Reply:
No width is needed. You’ll wrap the logo and nav in a header component that will have flex properties to push it left / right and also a max width of 1200, etc